MAINTAINERS.yml

```diff
@@ -2768,11 +2759,12 @@ NIOS-2 arch:
     - arch/nios2/
     - dts/nios2/intel/
     - boards/common/nios2.board.cmake
-    - boards/nios2/
-    - soc/nios2/
+    - soc/altera/zephyr_nios2f/
+    - soc/altera/qemu_nios2/
     - include/zephyr/arch/nios2/
     - tests/boards/altera_max10/
-    - boards/nios2/qemu_nios2/
+    - boards/qemu/nios2/
+    - boards/altera/altera_max10/
```

Comment:
boards/altera ?

Reply:
since this is scoped to nios2 I'd not generalize, there may be other non-nios2 altera boards